### DISCUSSION TITLE
Ladder Contributors

### DISCUSSION THREAD

**Discussion by mms-gianni** [kubero-dev/ladder#51] [Announcements]
@deoxykev @dxbednarczyk and @joncrangle 

I just want to thank you one more time, for the work and ideas you're adding to this project. 

I'm in contact with Kevin by e-mail. But to improve the communication between contributors I'd like to introduce discord channel, open for everyone but with a private section for contributors and project members.

https://discord.gg/brSHjrkE

**Comment by Klages** [comment]
Could you please post a new link? The one above is expired.

**Reply by dkbednarczyk** [reply]
Here is a permanent invite link: https://discord.gg/Fa8J5chHFQ

It has been quiet this year, but maybe once https://github.com/everywall/ladder/tree/origin/proxy_v2 gets merged there will be renewed interest in the project

@mms-gianni?
